src/Repositories/H5PContentRepository.php

```diff
@@ -252,4 +263,11 @@ private function getQueryContent(ContentFilterCriteriaDto $contentFilterDto, arr
 
         return $this->applyCriteria($query, $contentFilterDto->toArray());
     }
+
+    private function getUnused(): Collection
+    {
+        return H5PContent::query()
+            ->whereRaw('(SELECT COUNT(*) FROM topic_h5ps WHERE hh5p_contents.id = topic_h5ps.value) <= 0')
```

Comment:
jak będzie używana samodzielnie paczka do H5P to będzie to błąd

proponuje to zrobić tak jak zadanie #24 
https://github.com/EscolaLMS/H5P/blob/main/src/Traits/QueryExtendable.php
https://github.com/EscolaLMS/topic-types/blob/main/src/EscolaLmsTopicTypesServiceProvider.php

aby podczas dodawania topic-types było dodanane sprawdzanie czy istnieje powiązanie